third_party/accessibility/base/string_utils.cc

```diff
@@ -68,6 +118,18 @@ void ReplaceChars(std::string in,
   *out = in;
 }
 
+void ReplaceChars(std::u16string in,
+                  std::u16string from,
+                  std::u16string to,
+                  std::u16string* out) {
+  size_t pos = in.find(from);
+  while (pos != std::string::npos) {
+    in.replace(pos, from.size(), to);
+    pos = in.find(from, pos + to.size());
```

Comment:
nit: In theory `pos + to.size()` could overflow in a string exactly `npos` long with a last match that ends exactly at the end of the string... maybe? Actually, maybe not. I think the last possible match is `npos-to.size()` in which case your worst case is that this just sums to `npos`. I think I'm convinced, but you should double-check my math :)